src/Accounts/Accounts/Feedback/OpenAzSurveyLinkCmdlet.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Profile.Survey
+{
+    [Cmdlet(VerbsCommon.Open, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SurveyLink"), OutputType(typeof(void))]
+    public class OpenAzSurveyLinkCmdlet : AzureRMCmdlet
```

Comment:
Super class should be AzPSCmdlet

Reply:
any particular reason this has to be AzPSCmdlet?

Reply:
AzPSCmdlet is the super class of all cmdlets of Azure PowerShell. AzureRmCmdlet is one child class.